spec/unit/utils/validator.spec.js

```diff
@@ -1,62 +1,61 @@
-import assert from 'assert'
-import { domainFormat } from '../../src/renderer/utils/validator'
+import { domainFormat } from '@/utils/validator'
 
 describe('validator', () => {
   describe('domainFormat', () => {
     describe('string contains protocol', () => {
       const domain = 'https://mastodon.social'
       it('should not match', () => {
         const res = domain.search(domainFormat)
-        assert.strictEqual(res, -1)
+        expect(res).toEqual(-1)
       })
     })
     describe('string contains account name', () => {
       const domain = 'h3_poteto@mastodon.social'
       it('should not match', () => {
         const res = domain.search(domainFormat)
-        assert.strictEqual(res, -1)
+        expect(res).toEqual( -1)
```

Comment:
__[ESLint]__ <sub>posted by Sider</sub>

Error - There should be no spaces inside this paren. (space-in-parens)